app/controllers/taxa/names_controller.rb

```diff
@@ -200,4 +204,16 @@ def parse_classification_controller_params
     @move_to = params[:move_to]
   end
 
+  def limit_names
+    if current_user.newish? || (current_user.newish? && current_user.assistant_curator?)
+       names_count = Curator.total_objects_curated_by_action_and_user(Activity.add_common_name.id, current_user.id, [ChangeableObjectType.synonym.id],
+                             return_type = 'count', created_at = DateTime.now.to_date.beginning_of_day..DateTime.now.to_date.end_of_day)
```

Comment:
Useless assignment to variable - `return_type`.<br>Useless assignment to variable - `created_at`.<br>Line is too long. [136/80]<br>Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.